### DISCUSSION TITLE
Improvement suggestion for the documentation of Coil

### DISCUSSION THREAD

**Discussion by ibrahimyilmaz** [coil-kt/coil#905] [Ideas]
Hi @colinrtwhite 

First of all, thank you for `Coil`! It is a very very intuitive library to use and migrating to the coil would be one of the amazing decisions for Android Dev Teams.

Recently I do some trade-off analysis for image loading libraries. I am pretty sure that Coil covers the things as much as other libraries do(Bitmap Pooling, Memory Cache, Disk Cache, compression, etc).

Not only I but also the community will appreciate it if you put the information, which is about how you cover the things(like caching, Bitmap pooling, Disk caching) into the documentation of Coil.

#### Caching
|#| Resource|
|-|-|
|Fresco|https://frescolib.org/docs/caching.html|
|Glide|https://bumptech.github.io/glide/doc/caching.html|

I have found this URL for caching in Coil (https://coil-kt.github.io/coil/getting_started/#memory-cache) but it is not detailed as much as others.

It will be also nice to see : 
- How Coil stores the original image and downsampled images on the disk.
- How Coil handles the cleaning of the cache(What are the functions). 
- If Coil stores Bitmaps in GPU or not.
- How many disk cache Coil provides.
- and some other things which can be used in tradeoff analysis.

All the best
Ibrahim

**Comment by colinrtwhite** [comment]
Thanks, contributions welcome!
